src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteManyCommandResolver.java

```diff
@@ -63,6 +63,7 @@ private FindOperation getFindOperation(CommandContext commandContext, DeleteMany
         objectMapper,
         null,
         0,
-        0);
+        0,
+        false);
```

Comment:
Can we change here to use  FindOperation.unsorted() instead of direct constructor.

Reply:
Yes of course..

Reply:
Done.